pfcpiface/utils.go

```diff
@@ -33,14 +33,16 @@ func hex2int(hexStr string) uint32 {
 	return uint32(result)
 }
 
-func getOutboundIP(dstIP string) net.IP {
+func getOutboundIP(dstIP string) (net.IP, net.IP)  {
+	log.Println("SPGWC address IP: ", dstIP)
 	conn, err := net.Dial("udp", dstIP+":"+PFCPPort)
 	if err != nil {
 		log.Fatal(err)
 	}
 	defer conn.Close()
-
+    remoteAddr := conn.RemoteAddr().(*net.UDPAddr)
```

Comment:
Lot of editors have go formatting support. Can you please run it? It will take care of space/tab and alignment issues

Reply:
```
gofmt -s -w -l *.go
```